tests/connect/replicaset-no-replicaset-name-001.phpt

```diff
@@ -0,0 +1,42 @@
+--TEST--
+MongoDB\Driver\Manager: Connecting to Replica Set with an empty replicaSet argument
+--SKIPIF--
+<?php require __DIR__ . "/../utils/basic-skipif.inc"; ?>
+<?php skip_if_not_replica_set(); ?>
```

Comment:
Both errors (libmongoc URI parsing and our new exception) can be exercised without incurring I/O, so I think this test could be better rewritten like the existing `Manager::__construct()` error tests (e.g. [manager-ctor_error-003.phpt](https://github.com/mongodb/mongo-php-driver/blob/master/tests/manager/manager-ctor_error-003.phpt)). Note that the 003 test already handles the invalid type check, which you had duplicated above, so there's be no need to check that again in your new test.

With the SKIPIF removed, I think this would be as simple as two cases in their own `throws()` callable:

 * Invalid connection string with `?replicaSet=` throws libmongoc's exception
 * Valid connection string with `['replicaSet' => '']` URI options throws our exception